web_customisable_date_format/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# © 2016- Leonardo Donelli (donelli at monksoftware it)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Customisable date format',
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'category': 'Web',
+    'author': 'MONK Software',
```

Comment:
and also [here](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)

Reply:
OCA